### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

File: src/translations/main_nl.ts [hunk 1]
```diff
@@ -0,0 +1,946 @@
...
+        <source>Display the contents of the given mork database.</source>
+        <translation>Bekijk de inhoud van de opgegeven mork-databank.</translation>
+    </message>
+    <message>
+        <source>databaseFile</source>
+        <translation>databankbestand</translation>
+    </message>
+    <message>
+        <source>Decode an IMAP Utf7 string.</source>
+        <translation>Decodeer een IMAP-Utf6-string.</translation>
```
Comment (Abestanis) [inline src/translations/main_nl.ts]:
```suggestion
        <translation>Decodeer een IMAP-Utf7-string.</translation>
```
This doesn't look right.

Comment (Abestanis) [inline src/translations/main_nl.ts]:
```suggestion
        <translation>Decodeer een IMAP-Utf7-string.</translation>
```
This doesn't look right.

File: src/translations/installer_nl.ts [hunk 2]
```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl_NL">
+<context>
+    <name>WindowsInstaller</name>
+    <message>
+        <source>__LANGUAGE_NAME__</source>
+        <translatorcomment>This must be the name of one of the NSIS translation files at https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Contrib/Language%20files/ without the file extension.</translatorcomment>
+        <translation>Nederlands</translation>
```
[the remark this answers is not in the document]
Comment (Vistaus) [reply src/translations/installer_nl.ts]: You're right, I'll change that tomorrow!

File: src/translations/installer_nl.ts [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+        <source>TranslationsDescription</source>
+        <translation>Kies vertalingen voor ${PRODUCT_NAME}. ${PRODUCT_NAME} kiest automatisch een taal op basis van uw systeemtaal.</translation>
+    </message>
+    <message>
+        <source>UserSettingsSectionName</source>
+        <translation>${PRODUCT_NAME}-gebruikersinstellingen</translation>
+    </message>
+    <message>
+        <source>UserSettingsDescription</source>
+        <translation>Uw persoonlijke  ${PRODUCT_NAME}-instellingen en -configuratie.</translation>
```
[the remark this answers is not in the document]
Comment (Vistaus) [reply src/translations/installer_nl.ts]: Hmm, not sure what happened, but yes, that's unwanted.

File: src/translations/installer_nl.ts [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        <translation>De deïnstallatiewizard van ${PRODUCT_NAME} is afgebroken. Probeer het later opnieuw.</translation>
+    </message>
+    <message>
+        <source>Lang_en</source>
+        <translation>Engels</translation>
+    </message>
+    <message>
+        <source>Lang_de</source>
+        <translation>Duits</translation>
+    </message>
```
[the remark this answers is not in the document]
Comment (Vistaus) [reply src/translations/installer_nl.ts]: Will do so tomorrow :)

File: src/translations/main_nl.ts (hunk 1 above)
Comment (Vistaus) [reply src/translations/main_nl.ts]: Huh? I'm pretty sure I did not type a '6' there, but will fix that tomorrow then.

**Comment by Vistaus** [comment]
Pushed the requested changes, everything should look good now :)

**Comment by Abestanis** [comment]
Thank you, looks good to me.
@gyunaev Can I merge pull requests like these?

**Comment by gyunaev** [comment]
Yeah, please merge/commit directly those by yourself. Same goes to minor fixes/changes which are unlikely to break things.

**Comment by Abestanis** [comment]
@Vistaus A quick question: Do you want us to send you an email before we create a new Birdtray release? New translations are added all the time and old translation can change, while we are developing Birdtray. This would **not** mean that you are responsible for this translation, you are never forced to update them if you don't have the time or energy.

**Comment by Vistaus** [comment]
> @Vistaus A quick question: Do you want us to send you an email before we create a new Birdtray release?

Yes, please :)

**Comment by Abestanis** [comment]
Awesome, thank you! 👍

**Pull request merged**